modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2493,6 +2493,76 @@ protected ApiResponse findMethodResponse(ApiResponses responses) {
         return responses.get(code);
     }
 
+    /**
+     * Set op's returnBaseType, returnType, examples etc.
+     *
+     * @param operation endpoint Operation
+     * @param schemas a map of the schemas in the openapi spec
+     * @param op endpoint CodegenOperation
+     * @param methodResponse the default ApiResponse for the endpoint
+     */
+    protected void handleMethodResponse(Operation operation,
+                                     Map<String, Schema> schemas,
+                                     CodegenOperation op,
+                                     ApiResponse methodResponse) {
```

Comment:
All of this code was moved and not changed in any way

Reply:
Yup, noticed that too.